### PR TITLE
New shortly exercise

### PR DESCRIPTION
**Pull request by AminArria**

### CODE REVIEW

File: libraries/shortly/src/shortly_syn.erl [hunk 1]
```diff
@@ -0,0 +1,22 @@
+-module(shortly_syn).
+
+-behaviour(shortly_notification_algorithm).
+
+-export([init/0,
+         subscribe/1,
+         unsubscribe/1,
+         notify/1]).
+
+-define (SYN_NAME, syn_ws_connections).
```
Comment (unbalancedparentheses) [inline libraries/shortly/src/shortly_syn.erl]: remove space between define and (

File: libraries/shortly/Makefile [hunk 2]
```diff
@@ -10,3 +10,9 @@ test:
 	./rebar3 eunit
 	./rebar3 ct
 	echo "ALL TESTS PASSED"
```
Comment (unbalancedparentheses) [inline libraries/shortly/Makefile]: Mmm i don't know if I am a big fan of printing all test passed. I would remove this.

File: libraries/shortly/Makefile [hunk 3]
```diff
@@ -10,3 +10,9 @@ test:
...
+
+node_8010:
```
Comment (unbalancedparentheses) [inline libraries/shortly/Makefile]: rename. node 1, node 2 or node a, node b.

File: libraries/shortly/Makefile (hunk 2 above)
Comment (AminArria) [reply libraries/shortly/Makefile]: Yeah this should never be done. Also this target will be removed in favor of test targets only in the root Makefile

**Pull request merged**